### PR TITLE
go.mod: update secboot to eaa3655091fc

### PR DESCRIPTION
**Pull request by valentindavid**

### CODE REVIEW

**Review by pedronis — COMMENTED** [review]
are we already calling bootscope.SetModel? do we a TODO somewhere about that?

**Comment by valentindavid** [comment]
> are we already calling bootscope.SetModel? do we a TODO somewhere about that?

Yes, in `secboot_hooks.go`. For tpm is useless. There will be PR #14570 that will factor the code so that we will always have the same code path for both and we will always call SetModel.

File: secboot/secboot_hooks.go [hunk 1]
```diff
@@ -195,11 +195,6 @@ func unlockVolumeUsingSealedKeyFDERevealKeyV2(sealedEncryptionKeyFile, sourceDev
 
 	// the output of fde-reveal-key is the unsealed key
 	options := activateVolOpts(opts.AllowRecoveryKey)
-	// FIXME: consider setting it in activateVolOpts if we keep
-	// this function separate from the tpm one for key data in
-	// files. Otherwise we should only set it if we provide key
-	// data generation 1.
-	options.Model = model
 
 	sbSetModel(model)
```
Comment (valentindavid) [inline secboot/secboot_hooks.go:199]: Here. (with `sbSetModel = sb_scope.SetModel` earlier in the file)

**Review by pedronis — APPROVED** [review]
thanks

File: secboot/secboot_hooks.go (hunk 1 above)
Comment (bboozzoo) [inline secboot/secboot_hooks.go:199]: btw. is this covered by existing tests?

**Review by bboozzoo — APPROVED** [review]

File: secboot/secboot_hooks.go (hunk 1 above)
Comment (valentindavid) [reply secboot/secboot_hooks.go:199]: We do not verify it. I can add that in another PR.

Comment (bboozzoo) [reply secboot/secboot_hooks.go:199]: A followup PR is fine.

Comment (valentindavid) [reply secboot/secboot_hooks.go:199]: It was small. So I added it.

**Pull request merged**